pages/index.vue

```diff
@@ -4,12 +4,10 @@ await execute();
 </script>
 
 <template>
-  <UContainer>
+  <UContainer class="my-3">
     <Navigation></Navigation>
 
-    <header class="prose my-5">
-      <!-- <h1>{{ title }}</h1> -->
-    </header>
+    <header id="top" class="my-5"></header>
```

Comment:
Why not use an [anchor element](https://www.w3schools.com/tags/tag_a.asp#:~:text=If%20the%20%3Ca%3E%20tag%20has%20no%20href%20attribute%2C%20it%20is%20only%20a%20placeholder%20for%20a%20hyperlink.)? Let's also wrap the [LeaderSection Component](https://github.com/Commbocc/hc-county-leadership/blob/main/components/LeaderSection.vue#L6-L28) in a `section` element.